selectors/deletePage.selectors.ts

```diff
@@ -0,0 +1,5 @@
+export const deletePageSelectors =
+{
+    renameField: '#rename-field',
+    deleteConfirmationLabel: '[aria-label="Type in the name of the repository to confirm that you want to delete this repository."]',
```

Comment:
please use matcher for specific regix/string format.
ex: [aria-label^="repository to confirm that you want to delete"]